controllers/windowsmachine_controller.go

```diff
@@ -53,6 +53,8 @@ const (
 	MachineOSLabel = "machine.openshift.io/os-id"
 	// WindowsMachineController is the name of this controller in logs and other outputs.
 	WindowsMachineController = "windowsmachine"
+	// IgnoreLabel is a label that will cause machines to be ignored by the Windows Machine controller
+	IgnoreLabel = "windowsmachineconfig.openshift.io/ignore"
```

Comment:
You are couching this change as an `e2e` commit but you are actually changing the controller's behavior.

Reply:
Reworked the commit title to correct this

Reply:
Please document this.

Reply:
This has not been documented.

Reply:
Missed this, done